pkgs/tools/graphics/glee/default.nix

```diff
@@ -0,0 +1,34 @@
+{stdenv, fetchgit, pkgs}:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "glee";
+  rev = "f727ec7463d514b6279981d12833f2e11d62b33d";
+  version = "20170205-${stdenv.lib.strings.substring 0 7 rev}";
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://git.code.sf.net/p/${pname}/${pname}";
+    sha256 = "13mf3s7nvmj26vr2wbcg08l4xxqsc1ha41sx3bfghvq8c5qpk2ph";
+  };
+
+  nativeBuildInputs = with pkgs; [ cmake ];
+
+  buildInputs = with pkgs; [ mesa xorg.libX11 ];
+
+  configureScript = ''
+    cmake
+  '';
+
+  preInstall = ''
+    sed -i 's/readme/Readme/' cmake_install.cmake
```

Comment:
Do other folks run it on a case-insensitive filesystem or something? Seems weird, but makes sense